Packs/CortexXDR/ReleaseNotes/2_9_2.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Added the *status* integration parameter which enables you to filter fetched incidents by their status.
```

Comment:
also added status argument to the `xdr-get-incidents` command

Reply:
Done.